### PR TITLE
Info Popup(15 May)

### PR DESCRIPTION
**Pull request by Palakagrawal28**

### CODE REVIEW

**Comment by mohitsaladiya** [comment]
Please resolve the conflicts.

File: src/components/common/Modal/InfoModal/_InfoModal.scss [hunk 1]
```diff
@@ -12,16 +12,16 @@
 }
 
 .info-modal__content {
-  padding: 20px;
+  padding: 30px;
   margin-left: auto;
   margin-right: auto;
-  width: 30%;
-  height: 20%;
+  width: 40%;
```
Comment (amitjain24524) [inline src/components/common/Modal/InfoModal/_InfoModal.scss:18]: please set the width in PX for desktop(EX: 620px or  992px) an same for mobile (90%)

File: src/components/common/Modal/InfoModal/InfoModalFooter/_InfoModalFooter.scss [hunk 2]
```diff
@@ -4,16 +4,42 @@
...
+  margin: 30px 0;
   .info-modal-footer__button {
-    width: 160px;
-    height: 40px;
-    font-size: 18px;
-    padding: 0;
-    margin: 0 20px;
-    filter: brightness(1);
-    background: var(--secondary-color);
+    display: -webkit-box;
```
Comment (amitjain24524) [inline src/components/common/Modal/InfoModal/InfoModalFooter/_InfoModalFooter.scss:9]: use button component for footer button same css

File: src/components/common/Modal/InfoModal/InfoModalBody/_InfoModalBody.scss [hunk 3]
```diff
@@ -7,6 +7,9 @@
     font-size: 32px;
     line-height: 36px;
   }
+  font-size: 24px;
```
Comment (amitjain24524) [inline src/components/common/Modal/InfoModal/InfoModalBody/_InfoModalBody.scss:10]: already we have font-size, why we are adding same CSS double

**Pull request merged**